codemirror/codemirror.d.ts

```diff
@@ -181,7 +181,7 @@ declare namespace CodeMirror {
             handle: any;
             text: string;
             /** Object mapping gutter IDs to marker elements. */
-            gutterMarks: any;
+            gutterMarkers: any;
```

Comment:
Yes, I can see it's `gutterMarkers` in the actual source code:

https://github.com/codemirror/CodeMirror/blob/master/lib/codemirror.js#L5114

Well done! :+1: